src/runtime/cache.cpp

```diff
@@ -139,7 +144,7 @@ WEAK bool CacheEntry::init(const uint8_t *cache_key, size_t cache_key_size,
     for (size_t i = 0; i < key_size; i++) {
         key[i] = cache_key[i];
     }
-    for (int32_t i = 0; i < tuple_count; i++) {
+    for (uint32_t i = 0; i < tuple_count; i++) {
```

Comment:
Using "unsigned" (vs uint32_t) for loop counters may be slightly more performant on 64-bit architectures.